File: 3600-find-the-k-th-character-in-string-game-i/find-the-k-th-character-in-string-game-i.py
class Solution:
    def kthCharacter(self, k: int) -> str:
        str_1=[0]
        while(len(str_1)<k):
            for i in range(len(str_1)):
                str_1.append((str_1[i]+1)%26)
        
        print(str_1)
        return chr(str_1[k-1]+ord('a'))

File: 3600-find-the-k-th-character-in-string-game-i/find-the-k-th-character-in-string-game-i.java
class Solution {
    public char kthCharacter(int k) {
        StringBuilder str=new StringBuilder("a");
       str_manu(k,str);
       return str.charAt(k-1);
    }
    public void str_manu(int k,StringBuilder str)
    {
        if(str.length()>=k)
        {
            return;
        }
        int n=str.length();
        for(int i=0;i<n;i++)
        {
           char c=str.charAt(i);
           str.append((char)('a'+ ((c-'a'+1)%26)));
        }
        str_manu(k,str);
    }
}